src/components/Filter/modules/ListFilter/ListFilter.tsx

```diff
@@ -34,7 +34,7 @@ const ListFilter = (
     getFilterCount: (value) => value?.size ?? 0,
     getApiQueryUrl: (key, value) => {
```

Comment:
With the logic being modified here, there should be updated tests in the `listFilter.test.js` file to cover this.